src/displayapp/fonts/fix_jetbrains_mono_bold_20_zero.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
```

Comment:
Maybe we could use a normal patch file for patches.

Reply:
this will work only so long as the relevant lines stay in the same place, and look exactly the same (including the comment, which just changed between versions of lv_font_conv), which is why I made this over-complicated patcher script.

(I almost made it even more of a mess, by allowing for space changes and such).

so I recommend to leave it like this, but this really is a mess - and I'll abide by your decision.

Reply:
nvm about the comment (the diff does not need to include parts before and after), but the lines do still matter - which can change anytime a new symbol is added which ends up before.